[StyleCop] Fix warnings in BotBuilderRecognizerSamples

Bill7zz opened the pull request:
- Rename variables with Hungarian notation
- Add space after closing brace
- Document the return element in the method
- Reorder using directives
- Remove trailing whitespace

Aliandi committed:
Add the ignore of the XML warning

VictorGrycuk commented on .NET/Samples/BotBuilder/Dialogs/RootDialog.cs:
```diff
@@ -17,6 +17,13 @@ public async Task StartAsync(IDialogContext context)
 
+        private static string GetCurrentCultureCode()
+        {
+            // Use English as default culture since the this sample bot that does not include any localization resources
```
I think there is an extra `the` in the line `[...] since the this sample bot [...]`.

Aliandi replied:
@Bill7zz this is still here!

VictorGrycuk reviewed (approved):
LGTM!

Bill7zz committed:
Fix Stylecop warnings in SimpleConsole project
-XML comment analysis warning
-Update ParseAll method to return a list of model result

JuanAr reviewed (changes requested):
Feedback regarding commits

ceciliaavila reviewed (approved):
LGTM

Bill7zz committed:
Fix StyleCop warnings in BotBuilderSample project
-MessagesController
-AssemblyInfo
-WebApiApplication
-DeliveryPrompt
-QuantityPrompt
-RootDialo

-Add trailing coma in multi-line initializers
-Rename file Global to WebApiApplication
-Remove trailing whitespace

Aliandi commented on .NET/Samples/BotBuilder/BotBuilderRecognizerBot.csproj:
```diff
@@ -125,8 +125,8 @@
-    <Compile Include="Global.asax.cs">
-      <DependentUpon>Global.asax</DependentUpon>
+    <Compile Include="WebApiApplication.asax.cs">
+      <DependentUpon>WebApiApplication.asax</DependentUpon>
```
Ln 129, 128 and 117 arw your changes or the changes that came from master?

Bill7zz committed:
Fix typo in GetCurrentCultureCode method comment

JuanAr reviewed (approved):
LGTM
